MAINTAINERS.yml

```diff
@@ -3575,9 +3575,10 @@ Silabs Platforms:
   status: maintained
   maintainers:
     - jhedberg
+    - asmellby
   collaborators:
     - jerome-pouiller
-    - asmellby
+    - Martinhoff-maker
```

Comment:
@Martinhoff-maker and @asmellby are not a member of the project, see https://docs.zephyrproject.org/latest/project/project_roles.html#contributor

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/84921